persistence/soft-index/pom.xml

```diff
@@ -17,6 +17,11 @@
          <groupId>org.kohsuke.metainf-services</groupId>
          <artifactId>metainf-services</artifactId>
       </dependency>
+      <dependency>
+         <groupId>io.mashona</groupId>
+         <artifactId>mashona-logwriting</artifactId>
+         <version>1.0.0.Beta1</version>
```

Comment:
I think we still need the version to be defined in the `build-configuration/pom.xml` so that it can easily be substituted downstream. The dependency should probably be defined in the root pom's `dependencyManagment` section as well.

@pruivo Is this right?

Reply:
right, except the version/property is not mandatory to be in `build-configuration/pom.xml` since there is no plan to be consumed outside this repo.

Reply:
Ooops, I forgot to move this over. I was definitely planning on doing so.